test/conftest.py

```diff
@@ -135,12 +135,12 @@ def cql_test_connection(cql, request):
 def this_dc(cql):
     yield cql.execute("SELECT data_center FROM system.local").one()[0]
 
-# "test_keyspace" fixture: Creates and returns a temporary keyspace to be
+# "keyspace" fixture: Creates and returns a temporary keyspace to be
 # used in tests that need a keyspace. The keyspace is created with RF=1,
 # and automatically deleted at the end. We use scope="session" so that all
 # tests will reuse the same keyspace.
 @pytest.fixture(scope="session")
-def test_keyspace(cql, this_dc):
+def keyspace(cql, this_dc):
```

Comment:
I can't comment on the commit message so I'll do it here:
I'm scared of the comment on a clash with a fixture of cql-pytest, and also something that someone else said yesterday on the mailing list. @kostja "promised" me that the top-level test/conftest.py, which I still think was a mistake, only takes effect for tests that "opt in" to use it, and cql-pytest, alternator, redis, scylla-gdb, and others, will NOT use it. Now it appears that they do. This should be fixed instead of working around the problems it causes.

Reply:
There is no actual name clash, I got mistaken. I updated the commit message.